src/patternfly/demos/MasterDetail/examples/MasterDetail.md

```diff
@@ -0,0 +1,401 @@
+  master-detail-template--description="Body text should be Red Hat Text Regular at 16px. It should have leading of 24px because of its relative line height of 1.5"
+  }}
+
+  {{#> page-main-section page-main-section--modifier="pf-m-no-padding"}}
+
+    <!-- Drawer -->
+    {{#> drawer
+      drawer--id=(concat master-detail-template--id '-drawer')
+      drawer--modifier="pf-m-inline-on-2xl"
+      drawer-panel--IsOpen="true"
+      }}
+
+      {{#> drawer-section}}
+        {{> master-detail-toolbar master-detail-toolbar--HasViewToggle="true"}}
+        {{#> divider divider--type="div"}}{{/divider}}
+      {{/drawer-section}}
+
+      {{#> drawer-main}}
+        {{#> drawer-content
+          drawer-content--modifier="pf-m-no-background"
+          drawer-content-body--modifier="pf-m-padding"
+          }}
+
+          {{> master-detail-card-view}}
+        {{/drawer-content}}
+
+        {{#> drawer-panel
+          drawer-panel--type="div"
+          drawer-panel--attribute=(concat 'id="' master-detail-template--id '-panel" aria-label="Panel"')
+          drawer-panel--modifier=""
+          drawer-panel--NoBody="true"
+          }}
+
+          {{#> drawer-panel-body}}
+            {{> master-detail-content-header
+              master-detail-content-header--title="Patternfly"
+              master-detail-content-header--sub-title="PatternFly elements"
+              }}
+          {{/drawer-panel-body}}
+
+          {{#> drawer-panel-body}}
+            {{> master-detail-content-body}}
+          {{/drawer-panel-body}}
+
+        {{/drawer-panel}}
+      {{/drawer-main}}
+    {{/drawer}}
+  {{/page-main-section}}
+{{/master-detail-template}}
+```
+
+```hbs title=Master-detail-card-simple-list isFullscreen
+{{#> master-detail-template
+  master-detail-template--id="master-detail-card-simple-list-example"
+  master-detail-template--title="Master detail, in card, simple list"
+  master-detail-template--description="Body text should be Red Hat Text Regular at 16px. It should have leading of 24px because of its relative line height of 1.5"
+  }}
+
+  {{#> page-main-section}}
+    {{#> card}}
+
+      <!-- Drawer -->
+      {{#> drawer
+        drawer--id=(concat master-detail-template--id '-drawer')
+        drawer--IsStatic="true"
+        }}
+
+        {{#> drawer-main}}
+          <!-- Content -->
+          {{#> drawer-content drawer-content--NoBody="true"}}
+            {{#> drawer-content-body drawer-content-body--modifier="pf-m-no-padding"}}
+              {{> master-detail-simple-list}}
+            {{/drawer-content-body}}
+          {{/drawer-content}}
+
+          <!-- Panel -->
+          {{#> drawer-panel
+            drawer-panel--type="div"
+            drawer-panel--attribute=(concat 'id="' master-detail-template--id '-panel" aria-label="Panel"')
+            drawer-panel--modifier="pf-m-width-75-on-xl"
+            drawer-panel--NoBody="true"
+            }}
+
+            <!-- Panel  -->
```

Comment:
These are showing up in the DOM not as comments

Reply:
Are you sure? I'm seeing html comments in the console.